docusaurus/android_versioned_docs/version-draft/01-basics/01-overview.mdx

```diff
@@ -0,0 +1,140 @@
+
+This is also built on top of the state library, and offers easy integration of Stream Chat into a Compose-based Android application. It's also highly modular and customizable.
+
+Check out the Compose implementation in action by trying the open-source [Compose UI Components Sample App](https://github.com/GetStream/stream-chat-android/tree/main/stream-chat-android-compose-sample).
+
+:::note
+Using our Compose SDK in an Android Views based app is fully supported. It could provide you an amazing opportunity to explore and adopt Compose.
+:::
+
+### The Right SDK for You
+
+If your use case requires a very high level of customization, runtime theming changes, replacing whole parts of the default UI or stateless components that don't rely on our ViewModels,
+the **Compose SDK is the way to go**. It offers deep customization through several layers - the theme, modifiers for components, bound and stateless component overloads, smaller reusable components and slot APIs.
+
+If, instead, you're looking for a simpler, less customizable SDK which currently offers slightly better performance in some scenarios or your project is limited by technology, then
+the **UI Components SDK is a great option**. It still offers a fair amount of customization through the theme and its attributes and some programmatic ways to customize the UI.
+
+Whichever SDK you choose, **we guarantee you'll be happy and we'll provide amazing support** with your integration, through detailed documentation, guides and direct support.
+
+## Upgrade and Versioning Strategy
+
+Our Android libraries do **not** follow semantic versioning. <!-- TODO Is this accurate? -->
+
+<!-- TODO Is this correct? We increase the minor version for breaking changes? -->
+We increase the minor version whenever breaking changes are introduced. Patch releases only contain smaller fixes and improvements. <!-- TODO Is this correct? -->
+
+You can see all the SDK changes in the [releases page](https://github.com/GetStream/stream-chat-android/releases), and you can also find the release notes of all past releases in our [CHANGELOG file](https://github.com/GetStream/stream-chat-android/blob/main/CHANGELOG.md). These will always highlight breaking changes.
+
+We also maintain a separate document, [DEPRECATIONS](https://github.com/GetStream/stream-chat-android/blob/main/DEPRECATIONS.md), which lists deprecated constructs in the SDK, with their expected time of further deprecations and eventual removal.
+
+:::info
+You can get notified of new releases by using the _Watch_ button in the [getstream/stream-chat-android](https://github.com/GetStream/stream-chat-android) repository. You can tweak your watch preferences to subscribe only to release events.
+:::
+
+### How Should I Specify My Dependency Version?
+<!-- TODO Double check if this is correct -->
```

Comment:
TODO

Reply:
Usually we ask customers use a **fixed** version to avoid any conflicts or unpredicted behaviour.